src/nuclio/projects/project/functions/functions.service.js

```diff
@@ -450,11 +450,13 @@
                     },
                     {
                         id: 'secret',
-                        name: 'Secret'
+                        name: 'Secret',
+                        tooltip: 'Managing sensitive objects'
```

Comment:
you don't use this property `tooltip`, why do you need it?

Reply:
This property is used for tooltip in dropdown